### PR TITLE
Beret armour now matches the security helmet

### PR DESCRIPTION
**Pull request by san7890**
## About The Pull Request
This PR edits the beret's melee armor value down from 40 to 35, matching the same value of helmets.
Why It's Good For The Game

It makes absolutely no sense for a fancy cap made out of fabric to be STRONGER than a hard helmet.

## Changelog

:cl:
fix: Nanotrasen has now stopped including a big chunk of iron in their berets, making it a bit weaker to melee damage. Still just as fashionable, though.
/:cl:

### CODE REVIEW

**Comment by SuperNovaa41** [comment]
This is balance not a fix

**Comment by san7890** [comment]
> This is balance not a fix

ok

**Comment by optimumtact** [comment]
I'm not going to merge this until it removes all armour from berets, they're cloth hats

**Comment by o0cyann0o** [comment]
lame and stinky

**Comment by SuperNovaa41** [comment]
> > I'm not going to merge this until it removes all armour from berets, they're cloth hats
> 
> ok

look what you did

**Comment by san7890** [comment]
> > > I'm not going to merge this until it removes all armour from berets, they're cloth hats
> > 
> > 
> > ok
> 
> look what you did

this is the result of a discussion i had with the man

**Comment by qbmax32** [comment]
what’s the issue if they have the same armor value as their “realistic” equivalents ie: sec beret has same armor as sec helmet? It’s just cosmetic at that point.

**Comment by Fikou** [comment]
grief pr

**Comment by optimumtact** [comment]
The issue is that it doesn't make sense and the fridge logic of it annoys me

**Comment by o0cyann0o** [comment]
why did you change the description of the cc beret

**Comment by qbmax32** [comment]
Also this removes armor on hos beret too which is dumb because it’s a unique head of staff item, it should keep its armor

**Comment by san7890** [comment]
> why did you change the description of the cc beret

because it makes sense

**Comment by castawaynont** [comment]
Also remove armour off sec baseball caps and such

File: code/modules/clothing/head/jobs.dm [hunk 1]
```diff
@@ -242,12 +241,11 @@
 
 /obj/item/clothing/head/beret/sec
 	name = "security beret"
-	desc = "A robust beret with the security insignia emblazoned on it. Uses reinforced fabric to offer sufficient protection."
+	desc = "A robust (in looks) beret with the security insignia emblazoned on it."
 	icon_state = "beret_badge"
 	greyscale_config = /datum/greyscale_config/beret_badge
 	greyscale_config_worn = /datum/greyscale_config/beret_badge/worn
 	greyscale_colors = "#972A2A#F2F2F2"
-	armor = list(MELEE = 40, BULLET = 30, LASER = 30, ENERGY = 40, BOMB = 25, BIO = 0, FIRE = 20, ACID = 50, WOUND = 4)
```
Comment (optimumtact) [inline code/modules/clothing/head/jobs.dm:250]: keep the armour, but reduce the values by half for all of it.

File: code/modules/clothing/head/jobs.dm [hunk 2]
```diff
@@ -242,12 +241,11 @@
 
 /obj/item/clothing/head/beret/sec
 	name = "security beret"
-	desc = "A robust beret with the security insignia emblazoned on it. Uses reinforced fabric to offer sufficient protection."
+	desc = "A robust (in looks) beret with the security insignia emblazoned on it."
```
Comment (optimumtact) [inline code/modules/clothing/head/jobs.dm:244]: undo this change

File: code/modules/clothing/head/jobs.dm [hunk 3]
```diff
@@ -359,7 +355,7 @@
 
 /obj/item/clothing/head/beret/centcom_formal
 	name = "\improper CentCom Formal Beret"
-	desc = "Sometimes, a compromise between fashion and defense needs to be made. Thanks to Central Command's most recent nano-fabric durability enhancements, this time, it's not the case."
+	desc = "Sometimes, a compromise between fashion and defense needs to be made. Thanks to Central Command's nano-fabric durability enhancements, it's not the case."
```
Comment (optimumtact) [inline code/modules/clothing/head/jobs.dm]: why are you even touching this?

Comment (san7890) [reply code/modules/clothing/head/jobs.dm]: It made sense at the time when everything was having armor removed since the old text was "most recent", because saying "most recent" when no other beret had armor didn't really feel correct to me personally. I rewrote it a little bit, but given the other recommended changes I'll probably revert it.

**Comment by Nexofas** [comment]
> > why did you change the description of the cc beret
> 
> because it makes sense

Nanotech berets at 2500 makes sense. Berets are already inferior to helmets statwise. Leave berets alone so that some people can go fashionsouls.

**Comment by optimumtact** [comment]
If you feel up to it, it would be nice to have a mid game tech tree unlock that unlocks a steel fibre thread that can be used to patch the beret to double it's armour again.

**Comment by lemshake** [comment]
Kevlar is not a real material.

**Comment by optimumtact** [comment]
>Nanotech berets at year 2500 makes sense. Berets are already inferior to helmets statwise. Leave berets alone so that some people can go fashionsouls.

askign people to make a tradeoff if they wish to look fashionable is acceptable, if the beret was wanted as the standard item, then the helmet should have been resprited.

File: code/modules/clothing/head/jobs.dm [hunk 4]
```diff
@@ -339,12 +336,11 @@
 
 /obj/item/clothing/head/beret/durathread
 	name = "durathread beret"
-	desc = "A beret made from durathread, its resilient fibres provide some protection to the wearer."
+	desc = "A beret made from durathread. How fashionable!"
 	icon_state = "beret_badge"
 	greyscale_config = /datum/greyscale_config/beret_badge
 	greyscale_config_worn = /datum/greyscale_config/beret_badge/worn
 	greyscale_colors = "#C5D4F3#ECF1F8"
-	armor = list(MELEE = 15, BULLET = 5, LASER = 15, ENERGY = 25, BOMB = 10, BIO = 0, FIRE = 30, ACID = 5, WOUND = 4)
```
Comment (optimumtact) [inline code/modules/clothing/head/jobs.dm:347]: as long as the values here are under the values provided by the sec badge, this is fine.

File: code/modules/clothing/head/jobs.dm [hunk 5]
```diff
@@ -339,12 +336,11 @@
 
 /obj/item/clothing/head/beret/durathread
 	name = "durathread beret"
-	desc = "A beret made from durathread, its resilient fibres provide some protection to the wearer."
```
[the remark this answers is not in the document]
Comment (san7890) [reply code/modules/clothing/head/jobs.dm:342]: Do we want this to have armor? It doesn't make sense to say "some protection to the wearer" if we aren't going to give it armor.

**Comment by TheBonded** [comment]
> The issue is that it doesn't make sense and the fridge logic of it annoys me

@optimumtact  then make sure to remove armour from the trenchcoats and winter coats too, they're also "just fabric." Obviously it's impossible to create a lightweight armour plating that you can put under fabric. Never been done in real life, and kevlar is a lie invented by security members to sell more AP 9mm magazines.

 Teleportation, magic glowy dust rocks, laser swords and time travel are all within the logic of the universe, but putting cloth over armour plates or using synthetic fibers woven into cloth are just too far fetched ideas to ever be realistic I guess.

**Comment by optimumtact** [comment]
I"m really not interested in having a discussion with reddit.

**Comment by optimumtact** [comment]
Actually
>If you feel up to it, it would be nice to have a mid game tech tree unlock that unlocks a steel fibre thread that can be used to patch the beret to double it's armour again.

is a hard requirement, so if you dont' want to do the extra coding work, then just nerfing it to match the helmet is likely the best approach

let me know

**Comment by tralezab** [comment]
this was silly, there armor back in line and some weird desc changes. ogey
